Docs/BazelForiOSDevelopers.md

```diff
@@ -0,0 +1,133 @@
+# Bazel for iOS Developers
+
+This document is an introduction to Bazel for iOS developers. There's a few
+other resources available for Bazel as well:
+- https://docs.bazel.build/versions/master/tutorial/ios-app.html
```

Comment:
I would explain why you might want to read the readme instead of the link (this explains more of the why than the how)